### PR TITLE
home-assistant: 2020.12.1 -> 2020.12.2

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/home-assistant/core/releases/tag/2020.12.2

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/HAP-python/default.nix [hunk 1]
```diff
@@ -1,16 +1,16 @@
 { lib, buildPythonPackage, fetchFromGitHub, isPy3k, curve25519-donna, ed25519
-, cryptography, ecdsa, zeroconf, pytest }:
+, cryptography, ecdsa, zeroconf, pytestCheckHook }:
```
Comment (thiagokokada) [inline pkgs/development/python-modules/HAP-python/default.nix:2]: TIL, very nice :+1: .

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 107975` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python39Packages.denonavr</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python37Packages.HAP-python</li>
    <li>python37Packages.denonavr</li>
    <li>python38Packages.HAP-python</li>
    <li>python38Packages.denonavr</li>
    <li>python39Packages.HAP-python</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]

**Comment by mweinelt** [comment]
> `denonavr` doesn't build with Python 3.9 but builds with older Pythons. Looks like an upstream issue, should we report it and disable the build of `denonavr` on Python 3.9 thinking ?

Thanks reported in https://github.com/scarface-4711/denonavr/issues/179.

**Review by prusnak — APPROVED** [review]

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
https://github.com/NixOS/nixpkgs/pull/107975

1 package failed to build:
python39Packages.denonavr

6 packages built:
home-assistant python37Packages.HAP-python python37Packages.denonavr python38Packages.HAP-python python38Packages.denonavr python39Packages.HAP-python
```

**Comment by scarface-4711** [comment]
> 
> 
> > `denonavr` doesn't build with Python 3.9 but builds with older Pythons. Looks like an upstream issue, should we report it and disable the build of `denonavr` on Python 3.9 thinking ?
> 
> Thanks, reported in [scarface-4711/denonavr#179](https://github.com/scarface-4711/denonavr/issues/179).

Fixed, please use `denonavr==0.9.10`

**Comment by thiagokokada** [comment]
@scarface-4711 Thanks, I opened a PR: https://github.com/NixOS/nixpkgs/pull/108326

**Pull request merged**